vsphere/resource_vsphere_resource_pool.go

```diff
@@ -105,6 +110,12 @@ func resourceVSphereResourcePool() *schema.Resource {
 			Optional:    true,
 			Default:     -1,
 		},
+		"scale_descendants_shares": {
+			Type:         schema.TypeString,
+			Description:  "Determines if the shares of all descendants of the resource pool are scaled up or down when the shares of the resource pool are scaled up or down.",
+			Optional:     true,
```

Comment:
Same as above, let's set the default to `Disabled`. Unless `Disabled` is semantically different than "unset"? You mention a default in the docs though.

Reply:
I don't follow with that comment I apologize, did you mean, is it necessary to codify the default client side? Generally yes I think it's a good idea, that way the field is never `computed` it's entirely known at time of plan, which is generally better for Terraform.

Reply:
I am leaning on you though, is V6 of vSphere, is this behavior effectively "disabled" is disabled semantically the equivalent to blank?

Reply:
It v6 the capability would not have been present, but based on the above I do see value in explicitly setting the defaults.

Reply:
Set as:

```go
Default:      string(types.ResourceConfigSpecScaleSharesBehaviorDisabled),
```